hosting/docker-compose.test.yaml

```diff
@@ -45,3 +45,11 @@ services:
       - 6379
     healthcheck:
       test: ["CMD", "redis-cli", "ping"]
+
+  postgres:
```

Comment:
This will be added to all the tests, I am looking for a way to spin up this in a smarter way